packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -84,6 +84,12 @@ export interface InstanceProps {
    * @default - true
    */
   readonly deleteAutomatedBackups?: boolean;
+  /**
```

Comment:
Missing empty line before this line:

```suggestion

/**
```